### PR TITLE
Dockerfile created

### PR DESCRIPTION
**Pull request by thinkingfish**

### CODE REVIEW

**Comment by thinkingfish** [comment]
image now available at https://hub.docker.com/r/thinkingfish/pelikan/

I'm looking into how to turn it into an official image, not just under my username.

**Comment by sagar0** [comment]
awesome!

File: packages/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ENV PELIKAN_VERSION 0.1.2
+ENV PELIKAN_URL https://github.com/twitter/pelikan/archive/0.1.2.tar.gz
+ENV PELIKAN_SHA1 b8cb249fc44dc1a16f8ed9ce0be008c43ceb3599
+
+# build and install binaries, config
+RUN BUILDDEPS='cmake curl gcc libc6-dev make' \
+	&& set -x \
+	&& apt-get update && apt-get install --no-install-recommends -y $BUILDDEPS \
+	&& rm -rf /var/lib/apt/lists/* \
+	&& curl --insecure -SL "$PELIKAN_URL" -o pelikan.tar.gz \
```
Comment (seppo0010) [inline packages/docker/Dockerfile]: I assume you needed the `--insecure`, but it doesn't sound like a great solution. 😢

Comment (seppo0010) [inline packages/docker/Dockerfile]: Ah, you are doing the sha comparison, I'm ok with this then

**Pull request merged**